Engines/Wine/Verbs/DXVK/script.js

```diff
@@ -30,37 +28,36 @@ class DXVK {
 
         return this;
     }
-
+  
     go() {
         const wizard = this.wine.wizard();
         const prefixDirectory = this.wine.prefixDirectory();
         const sys32dir = this.wine.system32directory();
         const architecture = this.wine.architecture();
 
         print("NOTE: wine version should be greater or equal to 3.10");
-
-        if (operatingSystemFetcher.fetchCurrentOperationSystem() != "Linux") {
-            wizard.message(
-                tr(
-                    "DXVK might not work correctly on macOS. This is depending on Metal api support and MoltenVK compatibility layer advancement"
-                )
-            );
-        } else {
-            wizard.message(
-                tr(
-                    "Please ensure you have the latest drivers (418.30 minimum for NVIDIA and mesa 19 for AMD) or else DXVK might not work correctly."
-                )
-            );
+  
+        if (operatingSystemFetcher.fetchCurrentOperationSystem().getFullName() !== "Linux")
+        {
+            const answer = wizard.menu(
+                tr("DXVK is currently unsupported on non-Linux operating systems due to MoltenVK implementation being incomplete. Select how do you want to approach this situation.",
+                ["YES, continue with DXVK installation regardless", "NO, quit script alltogether", "Exit DXVK Installer, but continue with the script"]
+             );
+             
+             if (answer.text === "Exit DXVK Installer, but continue with the script") {
+                return this;
+             }
+             
+             if (answer.text === "NO, quit script alltogether" || !answer) {
+                throw new Error("User aborted the script.");
+             }
+        }
+        else {
+            this.wizard().message(tr("Please ensure you have the latest drivers (418.30 minimum for NVIDIA and mesa 19 for AMD) or else DXVK might not work correctly."));
```

Comment:
Please use the above defined constant `wizard` instead of `this.wizard()`. It is easier to read and more uniform this way

Reply:
ok